scripts/generator/graphql_generator/csharp/SDK/Serializable.cs.erb

```diff
@@ -6,7 +6,7 @@ namespace <%= namespace %>.SDK {
     using UnityEngine;
 
     public class Serializable {
-        public string ToJsonString() {
+        override public string ToString() {
```

Comment:
As part of the JSON refactor work, I'm moving `Serializable` to be an interface so we won't be able to override `ToString`. It would also be odd for implementors of `Serializable` to need to override `ToString`. Could we change this back to using `toJsonString`?